homeassistant/components/eq3btsmart/manifest.json

```diff
@@ -2,8 +2,8 @@
   "domain": "eq3btsmart",
   "name": "EQ3 Bluetooth Smart Thermostats",
   "documentation": "https://www.home-assistant.io/integrations/eq3btsmart",
-  "requirements": ["construct==2.10.56", "python-eq3bt==0.1.11"],
+  "requirements": ["construct==2.10.56", "python-eq3bt==0.2"],
   "codeowners": ["@rytilahti"],
```

Comment:
This should probably add `bluetooth` as a dependency in case they have removed it from their `default_config`

Reply:
I don't see any other components defining it as their dependency, but adding `dependencies: ["bluetooth"]` should do the trick, right?

Reply:
Yes.  Should be able to copy from `switchbot`.  If its not in there I messed up something

Reply:
:+1: my HEAD was in the past, and I can see it on switchbot, thanks.